### PR TITLE
[1.2.0] Updated README

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

**Review by utku-caglayan — APPROVED** [review]
Looks great

File: README.md [hunk 1]
```diff
@@ -18,40 +18,41 @@ import (
...
 
 ## Features
 
 * Distributed, partitioned and queryable in-memory key-value store implementation, called Map.
-* Additional data structures and simple messaging constructs such as Replicated Map, Queue, List, PNCounter, Set, Topic and others.
+* Additional data structures and simple messaging constructs such as Replicated Map, MultiMap, Queue, List, PNCounter, Set, Topic and others.
 * Support for serverless and traditional web service architectures with Unisocket and Smart operation modes.
 * Go context support for all distributed data structures.
 * Hazelcast Cloud integration.
+* SQL support (only on Hazelcast 5.x).
```
Comment (srknzl) [inline README.md:55]: Why don't we support 4.2 SQL?

File: README.md [hunk 2]
```diff
@@ -61,39 +62,39 @@ func main() {
 
 Requirements:
 
-* Hazelcast Go client is compatible only with Hazelcast IMDG 4.x and above.
-* We support Go 1.15 and up.
+* Hazelcast Go client is compatible only with Hazelcast 4.x and 5.x.
```
Comment (srknzl) [inline README.md:65]: Don't we need to add 6.x here once it is released 😀 ? Maybe 4.x and above is better.

File: README.md [hunk 3]
```diff
@@ -61,39 +62,39 @@ func main() {
...
+Check out our [Get Started](https://hazelcast.com/get-started/) page for options.
 
-The quickest way to start a single member cluster for development purposes is to use our Docker images.
+### Starting the Client with Hazelcast Cloud
 
-```
-docker run --rm --name hazelcast -p 5701:5701 hazelcast/hazelcast:4.2
-```
+You only need the cluster name and Hazelcast cloud token to start the client.
+If you haven't already, you can [http://cloud.hazelcast.com/sign-up](sign up) for a free Hazelcast Cloud account.
```
Comment (srknzl) [inline README.md:83]: Link and text should be swapped here. 
```suggestion
If you haven't already, you can [sign up](http://cloud.hazelcast.com/sign-up) for a free Hazelcast Cloud account.
```

File: README.md [hunk 4]
```diff
@@ -61,39 +62,39 @@ func main() {
...
-Hazelcast IMDG cluster also offers resilience.
-Should any hardware or software problem causes a crash to any member, the data on that member is recovered from backups and the cluster continues to operate without any downtime.
+Check out our [Get Started](https://hazelcast.com/get-started/) page for options.
 
-The quickest way to start a single member cluster for development purposes is to use our Docker images.
+### Starting the Client with Hazelcast Cloud
 
-```
-docker run --rm --name hazelcast -p 5701:5701 hazelcast/hazelcast:4.2
-```
```
Comment (srknzl) [inline README.md:89]: Do we want to discourage docker usage? We may delete it from here but I could not see docker mentioned in  https://hazelcast.com/get-started/ and https://hazelcast.com/open-source-projects/downloads/ as well

File: README.md (hunk 1 above)
Comment (yuce) [reply README.md:55]: SQL was beta in 4.2, so we only support SQL for 5.0. We can talk about the decisions offline.

File: README.md (hunk 2 above)
Comment (yuce) [reply README.md:65]: 6.x is a long way :) I specifically wanted to mention 5.x.

File: README.md (hunk 4 above)
Comment (yuce) [reply README.md:89]: We don't discourage Docker, but testing the client with Hz Cloud is easier for everyone.

File: README.md (hunk 3 above)
Comment (yuce) [reply README.md]: Fixed at: https://github.com/hazelcast/hazelcast-go-client/pull/710/commits/dc677bec1534ce600c73b48c17ee49fadb3e7eae

**Review by srknzl — APPROVED** [review]

**Pull request merged**